src/test/scala/chiselTests/CompatibilityInteroperabilitySpec.scala

```diff
@@ -390,4 +390,161 @@ class CompatibilityInteroperabilitySpec extends ChiselFlatSpec {
     compile(new Top(true))
     compile(new Top(false))
   }
+
+  "A Chisel.Bundle with only unspecified directions" should "work with D/I" in {
+    /* Fails on the instance <>:
```

Comment:
on my 3.5.x version of this I get a different error for this case, I get error in making the `Instance` itself that cannot mix specified and unspecified (the specified coming from Input(Clock) and Input(Reset)). Seeing if I need to refine my unit test or if this actually is not an issue on master due to fixes for Unspecified directions onmaster